doc/.ChangeLog_template

```diff
@@ -4,23 +4,12 @@ Originator(s):
 Date: 
 One-line Summary:
 
-Purpose of changes
-------------------
+Purpose and description of changes
```

Comment:
@billsacks thank you for making the change to this line. Now this raises a question (that I already wondered about before) about the line that says "Detailed list of changes" toward the bottom:

I take it to mean that we summarize the changes here and we describe them in detail near the bottom. I recommend more clarity about that if that's ok.

Reply:
Good point - that was confusing wording. Do you like my new wording better (that I just pushed)?

If you're happy with this, officially approving will help me track who is happy with the changes at this point :-)